src/js/components/componentDiagram/index.js

```diff
@@ -237,6 +252,12 @@ function renderGraph(componentDiagram) {
       const transform = e.parentElement.getAttribute('transform');
       const data = componentDiagram.graph.node(id);
 
+      // TODO:
+      // find a better and more consistent way to pass attributes
+      // to rendered element from corresponding node
+
+      // set parent-type data attribute
+      e.parentElement.setAttribute('data-parent-type', getParentType(id, componentDiagram.currentDiagramModel.class_package));
```

Comment:
This data is available in the model. In other words, is this class is a member of the 'HTTP' package?

Reply:
So we pass this data as attribute to svg element so it can be styled appropriately.

Reply:
So far, this code works. Maybe we can find a better solution here, but is this our priority right now?